docs/architecture/metadata/workflow.md

```diff
@@ -8,7 +8,7 @@ Metadata is added to ETL in the form of YAML files. We usually only add metadata
 ## Metadata in Snapshot
 The entrypoint to ETL is the Snapshot step. This is where we define metadata attributes for the data product of the origin. This is fundamental to ensure that the data is properly documented and that the metadata is propagated to the rest of the system.
 
-The metadata in Snapshot consists mainly of one object: `meta.origin`. To learn more about it, please refer to [the reference](../reference-origin).
+The metadata in Snapshot consists mainly of one object: `meta.origin`. To learn more about it, please refer to [the reference](../reference/origin).
```

Comment:
This link (and I think similar ones) lead me to `http://127.0.0.1:8000/architecture/workflow/snapshot/` (which yields a 404 error), but they should lead to `http://127.0.0.1:8000/architecture/metadata/reference-origin/`.